journal/202102.md

```diff
@@ -0,0 +1,399 @@
+# {DRAFT} Decred Journal – February 2020
+
+![{alt text}](../img/{file.jpg} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+{ introduction, major news of the month ordered by impact, don't put any links here }
+
+- { 1.6.1 }
+- { pos and pow upgraded, vote locked in }
+- { props approved }
+- { dcrdex 0.1.5 patch }
+- { kohola maybe? }
+
+## Release 1.6.1
+
+{ remind to verify }
+
+https://github.com/decred/decred-binaries/releases/tag/v1.6.1
+
+## Upcoming Consensus Change
+
+{ prepare to vote, set your prefs, dcrwallet }
+
+https://voting.decred.org/
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can build and run, but is not yet available in release binaries for regular users.
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+The v1.6.1 patch release included the following changes:
+
+- notification logic [modified](https://github.com/decred/dcrd/pull/2597) to force proof-of-work miners to upgrade so that voting on the new consensus changes can commence
+- fixed a rare [issue](https://github.com/decred/dcrd/pull/2582) where connections might not be reestablished after a network outage
+
+In development towards the next release, this month's highlight is the addition of a [UTXO cache](https://github.com/decred/dcrd/pull/2591) that cuts initial blockchain download (IBD) time by ~33% at the cost of some extra memory. It also significantly speeds up block processing during normal operation and reduces the wear and tear on the storage medium by allowing entries that are created and spent between flushing intervals to avoid be written at all. UTXO cache is a required step towards [multi-peer](https://github.com/decred/dcrd/issues/1145) parallel block downloading.
+
+Another highlight is the [introduction](https://github.com/decred/dcrd/pull/2579) of Age-Partitioned Bloom Filters. APBF is a lookup device that can quickly tell if it contains an element. It can track _a lot_ of data while using little memory, at the cost of a controlled rate of false positives. Unlike classic Bloom filters, it can handle an unbounded amount of data by aging and discarding old items. The motivation for adding ABPFs to Decred is to replace the LRU (least recently used) cache used to continuously track what data other peers are known to have. For a concrete example, the memory to track addresses known by 125 peers is reduced from ~200 to ~5 MiB when using ABPFs. Also, the package will be [useful](https://matrix.to/#/!zefvTnlxYHPKvJMThI:decred.org/$ulmCeMPuDSSmt3KTB09fJpypa4fpooFILTXKtQNWUCg) for other projects like wallet, DEX, mobile wallets, etc. Read the details in full glory in the [PR](https://github.com/decred/dcrd/pull/2579) and the [README](https://github.com/decred/dcrd/tree/master/container/apbf).
```

Comment:
There is a mix of ABPF and APBF used in this journal. I think they should all be APBF, and ABPF is erroneous

Reply:
Oops no idea how did I type ABPF so many times. Dem cosmic rays...